Core/OfficeDevPnP.Core/AppModelExtensions/FieldAndContentTypeExtensions.cs

```diff
@@ -739,6 +739,10 @@ public static void AddFieldToContentTypeByName(this Web web, string contentTypeN
         public static void AddFieldToContentType(this Web web, ContentType contentType, Field field, bool required = false, bool hidden = false)
         {
             contentType.EnsureProperties(c => c.Id, c => c.FieldLinks, c => c.SchemaXml);
```

Comment:
@SchauDK Would you properly load the Content Type, Field and FieldLinks instead?